### ISSUE TITLE
Arch Linux: cmark updated to 0.31.1

### ISSUE THREAD

**Issue by Anthringe** [PrismLauncher/PrismLauncher#2702]
### Operating System

Linux

### Version of Prism Launcher

8.4

### Version of Qt

Can't open launcher to check, however, system version of Qt is 6.7.2 and the launcher is built under qt6.

### Description of bug

Launcher will not start via shortcut or terminal

### Steps to reproduce

- Update cmark to 0.31.1 or later
- Attempt to start the launcher through a shortcut or terminal
- Nothing will happen (message will display if ran via terminal)

### Suspected cause

cmark was updated to 0.31.1 on archlinux, prismlauncher is still looking for "libcmark.so.0.31.0"

Upon trying to run the launcher via terminal, following message will display:

"prismlauncher: error while loading shared libraries: libcmark.so.0.31.0: cannot open shared object file: No such file or directory"

**Should** be fixable in the meantime by downgrading cmark, however, I have other packages that rely on cmark and don't want to downgrade in case any issues arise with those.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by rhysperry111** [comment]
What package are you using? As long as you're not using the `-bin` package you should just be able to rebuild to get Prism to link to the new library.

Also just as a PSA it seems that Prism might be coming to the official Arch repos soon.

EDIT: I don't personally use the `-bin` package, however after downloading the provided tar, `bin/prismlauncher` doesn't show up as linking to cmark anyway(?).

**Comment by Flimsy-Fox** [comment]
I have the the non-bin package installed, I had the same issue. Rebuilding the package fixed it.

**Comment by Anthringe** [comment]
Update: Thank you, rebuilding did fix it. Also, to the person who mentioned it, while I was looking at what packages depend on cmark, prismlauncher is now in Extra-Testing.

**Comment by K1ngjulien** [comment]
awesome, that fixed it for me as well! I've put a link to this thread in the comments of the AUR package.

with `yay` you can just install the package again and choose to do a clean build

```
...
AUR Explicit (1): prismlauncher-8.4-2
:: PKGBUILD up to date, skipping download: prismlauncher
  1 prismlauncher                    (Installed) (Build Files Exist)
==> Packages to cleanBuild?
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> 1
```

Out of curiosity, is there a way to handle this automatically, like linking to `libcmark.so` instead of `libcmark.so.0.31.0`? Or is that a bad practice?

**Comment by shardofgl4ss** [comment]
I fixed this at least temporarily by running  "sudo ln -s /usr/lib/libcmark.so.0.31.1 /usr/lib/libcmark.so.0.31.0" which just tricks Prism into using the updated one. No problems at all spoofing the library via symlink so far, without having to redownload/recompile PrismLauncher

Edit: This is definitely temporary cause if libcmark is updated again, youll have to sudo rm the symlink and make another to the updated one. However it works for me with no problems. A more permanent fix is to see if you can download libcmark 0.31.0 (doubtful since its a part of a larger package cmark) or to recompile Prism Launcher.

**Comment by GH0STP4WZ** [comment]
Sorry for necroposting but figured I'd bump this as I am currently experiencing this on my Arch Linux system. Currently recompiling the launcher again in hopes of resolving this, but if a solution could be found to this it would be nice.

Using [`prismlauncher-qt5`](https://aur.archlinux.org/packages/prismlauncher-qt5) from the AUR.
